### PR TITLE
add 'nested empty else' test to pretty-printer

### PR DESCRIPTION
**Pull request by reiner-dolp**

### CODE REVIEW

File: compiler-cli/integration-tests/ast/nested_empty_else.mj.stdout [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class A {
+	public void f() {
+		if (a)
+			if (b) { }
+			else ;
```
Comment (flip1995) [inline compiler-cli/integration-tests/ast/nested_empty_else.mj.stdout]: >  Falls der else-Teil eines if-Statements ein leeres Statement enthält, so wird der komplette else-Teil
nicht ausgegeben.

```suggestion
```

File: compiler-cli/integration-tests/ast/nested_empty_else.mj.stdout [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+		else i = 42;
```
Comment (flip1995) [inline compiler-cli/integration-tests/ast/nested_empty_else.mj.stdout]:
```suggestion
		else 
			i = 42;
```

**Comment by flip1995** [comment]
This is a specification problem, not a problem with our PP

**Comment by reiner-dolp** [comment]
It's a bug since the output has different semantics.

File: compiler-cli/integration-tests/ast/nested_empty_else.mj.stdout (hunk 1 above)
Comment (reiner-dolp) [reply compiler-cli/integration-tests/ast/nested_empty_else.mj.stdout]: That's exatcly what the test is trying to assert. It should NOT be removed.

**Comment by flip1995** [comment]
Hm. The fix would be to pretty print an if statement differently when the outer if statement has an else case.

**Comment by reiner-dolp** [comment]
Do we have test that ensures it still works for not nested if-else blocks? Otherwise LGTM. Can be merged.

**Pull request merged**